cirq/ops/pauli_gates_test.py

```diff
@@ -193,3 +193,13 @@ def test_apply_unitary():
     cirq.testing.assert_has_consistent_apply_unitary(cirq.X)
     cirq.testing.assert_has_consistent_apply_unitary(cirq.Y)
     cirq.testing.assert_has_consistent_apply_unitary(cirq.Z)
+
+
+def test_identity_multiplication():
+    a, b, c = cirq.LineQubit.range(3)
+    assert cirq.X(a) * cirq.I(a) == cirq.X(a)
+    assert cirq.X(a) * cirq.I(b) == cirq.X(a)
+    assert cirq.X(a) * cirq.Y(b) * cirq.I(c) == cirq.X(a) * cirq.Y(b)
+    assert cirq.I(c) * cirq.X(a) * cirq.Y(b) == cirq.X(a) * cirq.Y(b)
+    with pytest.raises(TypeError):
+        _ = cirq.H(c) * cirq.X(a) * cirq.Y(b)
```

Comment:
Add newline.

Reply:
Originally I did, but then the formatter complained.

> Changes in cirq/ops/pauli_gates_test.py require formatting:
> --- cirq/ops/pauli_gates_test.py	(original)
> +++ cirq/ops/pauli_gates_test.py	(reformatted)
> @@ -203,4 +203,3 @@
>      assert cirq.I(c) * cirq.X(a) * cirq.Y(b) == cirq.X(a) * cirq.Y(b)
>      with pytest.raises(TypeError):
>          cirq.H(c) * cirq.X(a) * cirq.Y(b)
> -        
> Some formatting needed on changed lines.
> The command "check/format-incremental" exited with 1.